src/components/Table/Table.tsx

```diff
@@ -201,13 +235,16 @@ const Table = ({
   expansionIconComponent,
   minWidthBreakpoint = 640,
   sortGroups = false,
+  showFooter = true,
```

Comment:
We can probably just replace this with a variable inside this component which checks each column to see if it has footer data